app/api/candidate/route.ts

```
@@ -0,0 +1,164 @@
+      },
+    });
+
+    if (!candidate) {
+      return NextResponse.json(
+        { error: "Candidate not found" },
+        { status: 404 }
+      );
+    }
+
+    return NextResponse.json(candidate);
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)